Breeze.UI/src/app/wallet/advanced/advanced.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { FormGroup, Validators, FormBuilder, AbstractControl } from '@angular/forms';
+import { NgbDateStruct } from '@ng-bootstrap/ng-bootstrap';
+
+import './monitor';
+import { AdvancedService } from './advanced.service';
+import { LoadingState } from './loadingState';
+import { SerialDisposable } from './serialDisposable';
+
+@Component({
+  selector: 'app-advanced',
+  templateUrl: './advanced.component.html',
+  styleUrls: ['./advanced.component.css']
+})
+export class AdvancedComponent implements OnInit, OnDestroy {
+    private addressCount = "";
```

Comment:
single quotes for consistency. My tslint plugin usually complains

Reply:
done